ui/src/store/axiosInstance.ts

```diff
@@ -31,4 +32,11 @@ axiosInstance.interceptors.request.use(async config => {
   return config;
 });
 
+axiosInstance.interceptors.response.use(
+  response => response,
+  error => {
+    throw handleApiError(error);
+  },
+);
+
```

Comment:
I am not sure we need to globally parse error since we are reducing the amount of information available for the rest of the application